src/server/tran_page_requests.hpp

```diff
@@ -31,14 +31,21 @@ enum class tran_to_page_request
   SEND_LOG_PRIOR_LIST,
 
   // Passive only
+  SEND_LOG_HEADER_LOG_APPEND_PREV_LSA_FETCH,
```

Comment:
The information required to boot the PTS log may be extended in the future. It is not feasible to maintain a name that includes all the transferred information.
```suggestion
  SEND_LOG_BOOT_INFO,
```

Reply:
Done